packages/sd-jwt/src/SdJwtOptions.ts

```diff
@@ -0,0 +1,42 @@
+import type { HashName, JwaSignatureAlgorithm } from '@aries-framework/core'
+import type { DisclosureFrame } from 'jwt-sd'
+
+export type SdJwtCreateOptions<Payload extends Record<string, unknown> = Record<string, unknown>> = {
+  holderDidUrl: string
+  issuerDidUrl: string
+  issuerOverrideJsonWebAlgorithm?: JwaSignatureAlgorithm
```

Comment:
`override` I added because it overrides the auto-detected jwa from the issuer did.
`issuer` because there is also a holder.

Will remove both, likely a bit too verbose here.